### ISSUE TITLE
conda

### ISSUE THREAD

**Issue by ehartford** [oh-my-fish/theme-agnoster#63]
please support anaconda

functions/fish_prompt.fish
```
if test "$CONDA_DEFAULT_ENV"
  set envs $envs "conda[$CONDA_DEFAULT_ENV]"
end
```

**Comment by ehartford** [comment]
I made a PR for this

https://github.com/oh-my-fish/theme-agnoster/pull/64

**Comment by ehartford** [comment]
Is this worth merging @sn0cr ?

**Comment by sn0cr** [comment]
Closed with #64 😉😄
